flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -22,7 +22,12 @@ class SentryFlutterOptions extends SentryOptions {
   /// Enable or disable the Crash handling on the Native SDKs, e.g.,
   /// UncaughtExceptionHandler and [anrEnabled] for Android.
   ///
-  /// Available only for Android.
+  /// SentryCrashIntegration (KSCrash) for iOS, turning this feature off on iOS
+  /// has a side effect which is missing the Device's context, e.g.,
+  /// App, Device and Operation System.
```

Comment:
@philipphofmann is there a workaround this or not possible right now?

Reply:
Not an easy one. We could extract some of the device context from SentryCrashIntegration, because it doesn't depend on SentryCrash. For most of the data, we need SentryCrash to be running. We would need to figure out if we can initialize SentryCrash without all the signal handlers and crash monitors. I was not able to figure this out after a short research.

Reply:
thanks, let's go this way for now and if this becomes an issue again, we see if we get this prioritized.